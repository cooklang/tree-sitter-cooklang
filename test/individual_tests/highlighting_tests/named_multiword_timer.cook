Let ~dough rest{30%minutes} then ~final proof{1%hour}.