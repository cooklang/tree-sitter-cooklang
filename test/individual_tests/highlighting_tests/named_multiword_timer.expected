(recipe
  (step
    (plain_text)
    (timer
      name: (timer_text)
      (amount
        (quantity)
        (units)))
    (plain_text)
    (timer
      name: (timer_text)
      (amount
        (quantity)
        (units)))))